db/schema.rb

```diff
@@ -291,6 +293,7 @@
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
     t.datetime "discarded_at"
+    t.json "privacy_policy_acceptance"
```

Comment:
Can you regenerate the schema so we only have the right changes please?

Reply:
PrivacyPolicy is now merged and this PR is now rebased :)